trace_decoder/src/compact/compact_prestate_processing.rs

```diff
@@ -684,10 +732,7 @@ impl<C: CompactCursor> WitnessBytes<C> {
         Ok((header, self.instrs))
     }
 
-    // TODO: Look at removing code duplication...
-    // TODO: Move behind a feature flag...
-    // TODO: Fairly hacky...
-    // TODO: Replace `unwrap()`s with `Result`s?
+    #[allow(dead_code)]
     fn process_into_instructions_and_keep_bytes_parsed_to_instruction_and_bail_on_first_failure(
```

Comment:
Do we plan on using it? Otherwise I'd just go for removing it

Reply:
no idea, maybe @BGluth can answer this, and the rest as well

Reply:
Gotta love my `88` character function names when I bring them out... :sunglasses:

So the intent here was a debugging tool that people could use who need to test their instruction output that they are generating. I started a PR a while back to move these behind feature flags, so I would say keep it for now. I'll deal with this right after SMT & continuation support.